themes/bootstrap3/templates/Recommend/Databases.phtml

```diff
@@ -1,7 +1,7 @@
 <div class="databases">
   <?php $databases = $this->recommend->getResults(); ?>
   <?php if (!empty($databases)): ?>
-    <h4><?=$this->transEsc('databases')?></h4>
+    <h2><?=$this->transEsc('databases')?></h2>
```

Comment:
So *this* heading is too long now, it wraps at full width.  Assuming @crhallberg wants to keep the font size as is, I will work with my librarian colleague for a shorter name.  I have some thoughts.

Reply:
I shortened it.

Reply:
While we're changing this, it occurs to me that it might be wise to rename the translation key as well, to something like `databases_recommend_heading` since `databases` is pretty vague. (I think I failed to catch this earlier because it started life as `eds_databases` which seemed better-differentiated, but then things evolved a bit from there).

Reply:
Agreed, done.